semgrep-core/src/parsing/other/yaml_to_generic.ml

```diff
@@ -148,7 +148,7 @@ let make_scalar _anchor _tag pos value env : A.expr =
     let token = mk_tok pos value env in
     match value with
     | "__sgrep_ellipses__" -> A.Ellipsis (Parse_info.fake_info "...")
-    | "null" | "NULL" | "" | "Null" | "~" -> A.L (A.Null token)
```

Comment:
This is just to merge this PR. I will put `""` this back to `Null` and have either a warning or an error when we try to parse a string and get Null.

Reply:
I am not very familiar with this code so I don't know exactly why this helps this PR, but I guess @aryx will know.